DuckDuckGo/TabSwitcherViewController+MultiSelect.swift

```diff
@@ -630,6 +510,18 @@ extension TabSwitcherViewController {
 // MARK: UIAction factories
 extension TabSwitcherViewController {
 
+    func action(title: String, _ handler: @escaping () -> Void) -> UIAction {
+        return UIAction(title: title) { _ in
+            handler()
```

Comment:
Using `handler` here causes strong retain cycle, leaving instances of `TabSwitcherViewController` and related classes in memory. You should capture weak self in all usage cases of `action` function.